src/components/Sources.css

```diff
@@ -59,13 +59,17 @@ html[dir="rtl"] .sources-header-info {
 .folder,
 .domain,
 .file,
-.worker {
+.worker,
+.refresh,
+.add-button {
   fill: var(--theme-splitter-color);
 }
 
 .domain,
 .file,
-.worker {
+.worker,
+.refresh,
+.add-button {
```

Comment:
hmm, perhaps we should have a separate SVG css file so that these styles are well grouped?

It's interesting you're running into a new use case for us. Which is not that un-common :)

yea, lets create a new file `utils/Svg.css` where we can add all the common SVG css. It can be a neighbor of https://github.com/devtools-html/debugger.html/blob/master/src/components/utils/Svg.js